Cargo.toml

```diff
@@ -17,13 +17,23 @@ log = "0.4"
 clap = "2.33.3"
 regex = "1"
 procfs = "0.8.1"
-actix-web = "3"
-riemann_client = "0.9.0"
+riemann_client = { version = "0.9.0", optional = true }
 hostname = "0.3.1"
 protobuf = "2.20.0"
-serde = { version = "1.0", features = ["derive"] }
-serde_json = "1.0"
-warp10 = "1.0.0"
+serde = { version = "1.0", features = ["derive"], optional = true }
+serde_json = { version = "1.0", optional = true }
+warp10 = { version = "1.0.0", optional = true }
 time = "0.2.25"
 colored = "2.0.0"
 chrono = "0.4.19"
+rs-docker = { version = "0.0.58", optional = true }
+hyper = { version = "0.14", features = ["full"], optional = true }
+tokio = { version = "1", features = ["full"]}
+#rs-docker = { path = "../rust-docker" }
+
+[features]
+default = ["prometheus", "riemann", "warp10", "containers", "json"]
```

Comment:
I'm not sure containers should be optional because at the end it will be in any exporter.

Reply:
I'm not pretty sure myself neither. Let's see if it's too mush pain or not to have it optional and make it not optional if needed.